Lib/pdb.py

```diff
@@ -1593,19 +1593,51 @@ def _runscript(self, filename):
 # Simplified interface
 
 def run(statement, globals=None, locals=None):
+    """Execute the statement (given as a string or a code object) under
+    debugger control.
+
+    The debugger prompt appears before any code is executed; you can set
+    breakpoints and type continue, or you can step through the statement
+    using step or next.
+
+    The optional globals and locals arguments specify the environment in
+    which the code is executed; by default the dictionary of the module
+    __main__ is used (see the explanation of the built-in exec() or
+    eval() functions.).
+    """
     Pdb().run(statement, globals, locals)
 
 def runeval(expression, globals=None, locals=None):
+    """Evaluate the expression (given as a string or a code object)
+    under debugger control.
+
+    When runeval() returns, it returns the value of the expression.
+    Otherwise this function is similar to run().
+    """
     return Pdb().runeval(expression, globals, locals)
 
 def runctx(statement, globals, locals):
     # B/W compatibility
     run(statement, globals, locals)
 
 def runcall(*args, **kwds):
+    """Call the function (a function or method object, not a string)
```

Comment:
```suggestion
    """Call the *function* (a function or method object, not a string)
```

Reply:
Thanks for the review! I went ahead and changed what you asked for except for this one case. I am not sure "function" should be italic as the name is not in the signature. What do you think?

Reply:
@carlbordum 

> Thanks for the review!

No problem, thanks for making the recommended changes.

> I am not sure "function" should be italic as the name is not in the signature. What do you think?

Personally, I'm a bit undecided on this one since it's in the function signature for the documentation: https://docs.python.org/3/library/pdb.html?highlight=runcall#pdb.runcall, I'm not certain why there's a discrepancy here between the docs and the source code. 

Perhaps someone experienced with pdb can provide clarification. In the meantime, it's probably fine to leave it out.

Reply:
The actual keyword is `func` https://docs.python.org/3/library/bdb.html#bdb.Bdb.runcall